charts/opentelemetry-collector/templates/_pod.tpl

```diff
@@ -11,7 +11,6 @@ containers:
     command:
       - /{{ .Values.command.name }}
       - --config=/conf/relay.yaml
-      - --metrics-addr=0.0.0.0:8888
```

Comment:
This should be defined in the default config service -> telemetry section. Can you please update the config accordingly?

Reply:
I’ve added the missing services. Checked that my prom instance scraping works as before.